src/lambda/steps/3_crawlPageAndQueueUrls.ts

```diff
@@ -51,7 +51,7 @@ export const crawlPageAndQueueUrls = async (
     }, destination);
     console.log('Synced content from', path);
 
-    console.log('Queueing', urlPaths.length, 'new urls to visit', urlPaths);
+    console.log('Queueing ', urlPaths.length, ' new urls to visit', urlPaths);
```

Comment:
The spaces are rendered automatically by `console.log` so you shouldn't need to add them :)

Reply:
Fixed